docs/getting-started/supported-languages-and-tools.md

```diff
@@ -340,6 +340,13 @@ The table below lists all languages and frameworks that Codacy supports and the
       <td><a href="https://github.com/kucherenko/jscpd">jscpd</a></td>
       <td><a href="https://eslint.org/">ESLint</a></td>
     </tr>
+    <tr>
+      <td>Unity</td>
```

Comment:
We should be saying that the language is C# and Unity is one of the "plugins" we are supporting currently.

Reply:
My idea was to associate the Rosnlyn Analyzers with the Unity framework (notice that on the [Supported languages and tools](https://feat-add-roslyn-tool-io-100--docs-codacy.netlify.app/getting-started/supported-languages-and-tools/) the first column is called "Language or framework" and that we have entries there like Kubernetes or Terraform). This would give more visibility to the fact that we support analyzing Unity code (in C#).

However, I made a new proposal - that I'm still not fully convinced about. :sweat_smile: What do you think?

Reply:
That doesn't seem wrong either, since the tool right now **only** supports unity rules, but the tool is generic enough to later be changed to include other rules. Like eslint that supports multiple plugins to support more stuff. 

I would maybe fall on the side of keeping it in the C# language since the tool will is associated to the C# language files so a user will see the tool recommended in the UI in the Code Patterns page when they simply have c# files... But I'm totally open to another approach here.

Reply:
Perhaps @fjrdomingues has an opinion on this.